rnndb/graph/gm200_texture.xml

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<database xmlns="http://nouveau.freedesktop.org/"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
+<import file="copyright.xml"/>
+
+<import file="nvchipsets.xml"/>
+<import file="g80_defs.xml"/>
+
+<enum name="TIC_HEADER_VERSION" inline="yes">
+	<value value="0x0" name="ONE_D_BUFFER"/>
+	<value value="0x1" name="PITCH_COLORKEY"/>
+	<value value="0x2" name="PITCH"/>
+	<value value="0x3" name="BLOCKLINEAR"/>
+	<value value="0x4" name="BLOCKLINEAR_COLORKEY"/>
+</enum>
+
+<domain name="TIC" size="0x20" prefix="chipset" variants="GM107-">
```

Comment:
I wonder if this should be called TIC2. On GM107 both this and the other one will be available, and I want to be able to include this file in rnndb/root.xml